stablehlo/dialect/Version.h

```diff
@@ -59,6 +60,14 @@ class Version {
 };
 
 mlir::Diagnostic& operator<<(mlir::Diagnostic& diag, const Version& version);
+llvm::raw_ostream& operator<<(llvm::raw_ostream& os, const Version& version);
+
+template <typename VersionedInterface>
+bool isLegalVersionForTarget(VersionedInterface& interface,
```

Comment:
Given that this function is only used in VhloToVersion.cpp, let's move it to that file.